pkg/mimirtool/config/descriptors/cortex-v1.11.0.json

```diff
@@ -8948,6 +8948,17 @@
               "fieldFlag": "store-gateway.sharding-ring.tokens-file-path",
               "fieldType": "string"
             },
+            {
+              "kind": "field",
+              "name": "num_tokens",
+              "required": false,
+              "desc": "Number of tokens for each store-gateway.",
+              "fieldValue": null,
+              "fieldDefaultValue": 512,
+              "fieldFlag": "store-gateway.sharding-ring.num-tokens",
+              "fieldType": "int",
+              "fieldCategory": "advanced"
+            },
```

Comment:
Did you add this manually? I don't think this change is necessary. This is the json descriptor of the cortex config as of version v1.11.0. Same for the other json descriptors

Reply:
Let me analyze what you pointed above and see if the imbalance is actually because of the block size rather than the number of blocks itself, I agree if this is due to the block size in difference then this PR will not do much good.

Reply:
The store-gateway which is almost running out of disk(98%) and the the one with the least disk space (65%) have a difference of 100 tokens within them, with the first one having 300 and the later having 200. Although the block size on average doesn't seem to be a problem averaging around 600-700 mb on both of them.

@dimitarvdimitrov These are the observations I made so far, what do you think?

Reply:
then I agree that changing the number of tokens makes sense 👍

Reply:
Perfect thanks @dimitarvdimitrov . I see the test failing seems like I will have to run the make reference command which I forgot to run before last commit